frontend/appflowy_flutter/lib/plugins/database_view/application/filter/filter_service.dart

```diff
@@ -84,10 +84,19 @@ class FilterBackendService {
     required String fieldId,
     String? filterId,
     required DateFilterConditionPB condition,
+    required FieldType fieldType,
     int? start,
     int? end,
     int? timestamp,
   }) {
+    assert(
+      [
+        FieldType.DateTime,
+        FieldType.UpdatedAt,
+        FieldType.CreatedAt,
+      ].contains(fieldType),
+    );
```

Comment:
In addition to the assert, I suggest returning it and adding an assert inside. This is because in release mode, the assert won't have any effect, and we should filter out incompatible parameters.

```dart
if (![...].contains(fileType)) {
  assert('some message').
  return;
}
```

Reply:
Maybe throw an exception? Because the return can't be empty.